### PR TITLE
add notes about part 2

### PR DESCRIPTION
**Pull request by AprilGar**

### CODE REVIEW

File: Building-A-RESTful-API-With-Scala-Play/Part2.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ In this section we'll give our API the ability to create, read, update and delet
 
 ## Create A New Controller
 1. Create a new Scala file in the controllers package called ApplicationController
+
+[//]: # (Can we not just say Scala class rather than file?)
```
Comment (vinniebrice) [inline Building-A-RESTful-API-With-Scala-Play/Part2.md:7]: Good point, I mention here scala file as filename.scala as anything can be in that (classes, objects, traits...) 
We could mention to them to make a new class however I wouldn't want them to make a class within the same file as another class.

This is so we can separate our concerns with code, this has many benefits down the line like readability, refactoring and adding enhancements as some files can remain untouched. 

In short, happy to say make a new class as long as its clear that it needs to be in a file in the right package

File: Building-A-RESTful-API-With-Scala-Play/Part2.md [hunk 2]
```diff
@@ -26,6 +29,8 @@ In this section we'll give our API the ability to create, read, update and delet
     ```
     
 5. Run your application and hit the new route. You should see the TODO text.
+
+[//]: # (This gave me a browser error where I needed to import Singleton. Add this?)
```
Comment (vinniebrice) [inline Building-A-RESTful-API-With-Scala-Play/Part2.md:33]: Yep! That needs to be added if its giving errors, thanks and good spot! Might be worth writing a note what singleton means. (Essentially it means only one instance of the class can be created at one time, preserves immutability and whatnot)
